dune-project

```diff
@@ -2,3 +2,30 @@
 (name lambdapi)
 
 (using menhir 1.0)
+
+(generate_opam_files true)
+(source (github Deducteam/lambdapi))
+(license CeCILL2.1)
+(authors "Rodolphe Lepigre <lepigre@mpi-sws.org>"
+         "Frédéric Blanqui <frederic.blanqui@inria.fr>"
+         "Emilio Jesús Gallego Arias <e@x80.org>")
+(maintainers "dedukti-dev@inria.fr")
+(package
+ (name lambdapi)
+ (synopsis "Proof assistant based on the λΠ-calculus modulo rewriting")
+ (description "Proof assistant based on the λΠ-calculus modulo rewriting,
+mostly compatible with Dedukti.
+
+This package contains both the `lambdapi` proof assistant and the `lp-lsp`
+Language Server Protocol helper for IDEs to check lambdapi files.")
+ (depends
+  (ocaml (>= 4.07.0))
+  (dune (>= 2.1.0))
```

Comment:
This is automatically added from the dune-lang version I think.

Reply:
Indeed it is, well seen.